packages/client/src/components/login-form.tsx

```diff
@@ -11,10 +11,12 @@ const Form = styled.form`
 
 export default function LoginForm() {
   return (
-    <Form action="/api/login" method="post">
-      <label>username</label> <input name="username" type="text" />
-      <label>password</label> <input name="password" type="password" />
-      <button type="submit">submit</button>
+    <Form action="/login" method="post">
+      <label>Username</label> <input name="username" type="text" />
+      <label>Password</label> <input name="password" type="password" />
+      <button type="submit">Submit</button>
```

Comment:
Avoid copy and pasting like this. Extract what is the same and keep apart everything that is different. 

We can reuse a lot of these components. See https://github.com/chewnoill/collaborative-editor/pull/44

Reply:
Just took a look. That looks awesome